scripts/west_commands/runners/jlink.py

```diff
@@ -337,6 +344,8 @@ def flash(self, **kwargs):
                     '-CommanderScript', fname] +
                    (['-nogui', '1'] if self.supports_nogui else []) +
                    self.tool_opt)
+            
+            self.logger.info('Command: {}'.format(cmd))
```

Comment:
Remove this; it's too verbose for the default behavior and it's already printed with `west -v flash` by `self.check_call`